src/Nethermind/Nethermind.Runner/Ethereum/Steps/StartMonitoring.cs

```diff
@@ -48,7 +48,7 @@ public async Task Execute(CancellationToken cancellationToken)
                 _api.LogManager.SetGlobalVariable("nodeName", metricsConfig.NodeName);
             }
             
-            if (metricsConfig.Enabled)
+            if (metricsConfig.PushEnabled || metricsConfig.ExposePort != null)
```

Comment:
&& or || ?
generally if enabled is false then we do not want to run it just because the port is set

Reply:
It's intentional - maybe it's not clear but `PushEnabled` only enables the pushing, while configuring `ExposePort` will start the listener. Either or both can be enabled.

Reply:
yes, understand now, thanks